### PR TITLE
test(wait-on): add automatic test to pre-push

### PR DESCRIPTION
**Pull request by jorgeucano**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jorgeucano** [comment]
oh and I fix the problem with one test in windows

File: src/__tests__/__snapshots__/blog-index.spec.ts.snap [hunk 1]
```diff
@@ -42,7 +42,7 @@ main[] {
...
...YTtBQUNmIiwiZmlsZSI6InByb2plY3RzL3NhbXBsZUJsb2cvc3JjL2FwcC9hcHAuY29tcG9uZW50LmNzcyIsInNvdXJjZXNDb250ZW50IjpbIjpob3N0IHtcclxuICBkaXNwbGF5OiBncmlkO1xyXG4gIGhlaWdodDogMTAwdmg7XHJcbiAgZ3JpZC10ZW1wbGF0ZS1yb3dzOiA2MHB4IDFmciA2MHB4O1xyXG59XHJcblxyXG46aG9zdCA+IGhlYWRlciB7XHJcbiAgZGlzcGxheTogZ3JpZDtcclxuICBiYWNrZ3JvdW5kLWNvbG9yOiByb3lhbGJsdWU7XHJcbiAgY29sb3I6IHdoaXRlc21va2U7XHJcbiAgbWFyZ2luOiAwO1xyXG4gIHBhZGRpbmc6IDBweCAxMHB4O1xyXG4gIGdyaWQtdGVtcGxhdGUtY29sdW1uczogMWZyIDQwcHg7XHJcbiAgcGxhY2UtaXRlbXM6IGNlbnRlciBjZW50ZXI7XHJcbn1cclxuXHJcbmhlYWRlciBoMSxcclxuZm9vdGVyIGgzIHtcclxuICBtYXJnaW46IDA7XHJcbn1cclxuOmhvc3QgPiBmb290ZXIge1xyXG4gIGRpc3BsYXk6IGdyaWQ7XHJcbiAgYmFja2dyb3VuZC1jb2xvcjogcm95YWxibHVlO1xyXG4gIGNvbG9yOiB3aGl0ZXNtb2tlO1xyXG4gIG1hcmdpbjogMDtcclxuICBwYWRkaW5nLXJpZ2h0OiAxMHB4O1xyXG4gIGp1c3RpZnktY29udGVudDogcmlnaHQ7XHJcbiAgYWxpZ24tY29udGVudDogY2VudGVyO1xyXG59XHJcblxyXG5tYWluIHtcclxuICBiYWNrZ3JvdW5kLWNvbG9yOiB3aGl0ZXNtb2tlO1xyXG4gIGNvbG9yOiByZ2IoNCwgMTIsIDM2KTtcclxuICBwYWRkaW5nOiAxMHB4O1xyXG59XHJcbiJdfQ== */</style><script charset=\\"utf-8\\" src=\\"blog-blog-module-es2015.js\\"></script><script charset=\\"utf-8\\" src=\\"pagenotfound-pagenotfound-module-es2015.js\\"></script><style>h1[] {
```
Comment (aaronfrost) [inline src/__tests__/__snapshots__/blog-index.spec.ts.snap]: We should probably clean the css sourcemaps up out of the snapshots. This is always going to change.

Comment (jorgeucano) [reply src/__tests__/__snapshots__/blog-index.spec.ts.snap]: yes, im updating, im not use the prod mode

File: src/__tests__/exclude.spec.ts [hunk 2]
```diff
@@ -2,16 +2,19 @@ import {readFileSync} from 'fs';
...
     let fileExists = true;
     // Trying to read the missing file should throw an error. If error doesn't happen, test fails. Meaning...
     // if the files exists (which it shouldn't, the test fails.
     try {
       readFileSync(join(__dirname, '../../dist/static/exclude/notpresent/index.html'), 'UTF8');
     } catch ({message, code, path}) {
-      if (code === 'ENOENT' && path && path.endsWith('/notpresent/index.html')) {
-        fileExists = false;
+      if (code === 'ENOENT' && path &&
+        ( path.endsWith('/notpresent/index.html')
```
Comment (aaronfrost) [inline src/__tests__/exclude.spec.ts]: @jorgeucano next time... use `path.sep`. It is the separator for whatever OS you're on. 
So you could do 
```
`${path.sep}notpresent${path.sep}index.html`
```
 to create the comparator. Then you don't have to have windows/linux shit in the code?

**Comment by SanderElias** [comment]
I can't commit my own changes now, so I'm going to merge this one now
@jorgeucano take note of Aarons remark.

**Pull request merged**